docs/docs/tutorials/wsl-can/index.md

```diff
@@ -0,0 +1,125 @@
+# :material-linux: Virtual CAN on WSL2
+
+By default, WSL2 does not support Virtual CAN, however it can be enabled by building a custom WSL2 kernel.
+
+!!! warning "This setup assumes you are running within WSL. If you do not have WSL installed, open Windows Powershell and run:"
+        wsl --install
+
+## Step 1: Update the WSL Environment
+
+Run the following commands in your WSL terminal to update your environment and install the required dependencies.
+
+    sudo apt-get update -y
+
+### Install dependencies:
+
+    sudo apt install -y dwarves libelf-dev flex bison libssl-dev libncurses-dev bc build-essential make
+    sudo apt install -y --no-install-recommends wslu
+
+## Step 2: Download the WSL2 Linux Kernel
+
+To ensure compatibility, you need the specific kernel version of WSL2.
+
+1. **Check your kernel version:**
+
+        uname -r
+
+    - Example output: `5.15.153.1-microsoft-standard-WSL2`
```

Comment:
is this the exact required kernel version? Or what is considered correct?

Reply:
I think you mean that the download has to match the `uname` output. Can you make this more clear?